highlight.io/components/QuickstartContent/backend/java/shared-snippets.tsx

```diff
@@ -0,0 +1,75 @@
+import { siteUrl } from '../../../../utils/urls'
+import { QuickStartStep } from '../../QuickstartContent'
+
+export const installSdk: QuickStartStep = {
+	title: 'Install the Highlight Java SDK.',
+	content: 'Add Highlight to your maven pom file.',
+	code: {
+		text: `<dependency>
+  <groupId>io.highlight</groupId>
+  <artifactId>highlight-sdk</artifactId>
+  <version>latest</version>
+</dependency>`,
+		language: 'xml',
+	},
+}
+
+export const initializeSdk: QuickStartStep = {
+	title: 'Initialize the Highlight Java SDK.',
+	content:
+		"`Highlight.init()` initializes the Highlight backend SDK.",
+	code: {
+		text: `HighlightOptions options = HighlightOptions.builder("PROJECT_ID")
```

Comment:
`"PROJECT_ID"` -> `"<YOUR_PROJECT_ID>"` 
that's the standard placeholder we replace on our `/setup` page

Reply:
Resolved in #[96d4453](https://github.com/highlight/highlight/pull/5286/commits/96d44539cbbbf8b8ca68d20f43de6180a48341ac)

Reply:
thanks!